ui/pages/Public/LandingPage.jsx

```diff
@@ -28,46 +38,50 @@ export default () =>
       Broad Institute.
-    </Segment>
-    <Segment size="large" padded="very">
-      <Header
-        textAlign="center"
-        size="medium"
-        content={
-          <span>
-            Currently, <i>seqr</i> is in closed beta-testing mode and is only used by our group and collaborators
-          </span>}
-      />
-      <Grid>
-        <Grid.Column width={3} />
-        <Grid.Column width={10}>
-          <List bulleted>
-            <List.Item>
-              If you are interested in collaborating with our group, please &nbsp;
-              <a href="mailto:seqr@broadinstitute.org"><b>contact us</b></a>
-            </List.Item>
-            <List.Item>
-              To get updates about seqr, &nbsp;
-              <a target="_blank" href="http://groups.google.com/a/broadinstitute.org/forum/#!forum/seqr-updates/join">
-                <b>join our mailing list</b>
-              </a>
-            </List.Item>
-            <List.Item>
-              The <a target="_blank" href="http://github.com/broadinstitute/seqr"><b>source code</b></a> is available on
-              github. Please use the &nbsp;
-              <a target="_blank" href="http://github.com/broadinstitute/seqr/issues"><b>github issues page</b></a> to
-              submit bug reports or feature requests
-            </List.Item>
-          </List>
-        </Grid.Column>
-        <Grid.Column width={3} />
-      </Grid>
-    </Segment>
+    </PageSegment>
+    <PageSegment size="large">
+      <Header textAlign="center" size="large"><i>seqr</i> is available through three methods:</Header>
+      <VerticalSpacer height={10} />
+      <List ordered>
+        <List.Item>
+          Available for all collaborators within the &nbsp;
+          <Anchor href="https://cmg.broadinstitute.org">Broad Institute Center for Mendelian Genomics</Anchor> or
+          Mendelian Genomics Research Center with data pre-loaded into projects
+        </List.Item>
+        <List.Item>
+          Available for use on the <Anchor href="https://anvilproject.org">AnVIL platform</Anchor> where requests
+          can be placed for loading a joint called vcf into seqr
+        </List.Item>
+        <List.Item>
+          Available on GitHub as an &nbsp;
+          <Anchor href="http://github.com/broadinstitute/seqr">open source project</Anchor> for download and local
+          installation
+        </List.Item>
+      </List>
+    </PageSegment>
+    <PageSegment secondary>
+      <List bulleted>
+        <List.Item>
+          If you are interested in collaborating with our group, please &nbsp;
+          <Anchor href="mailto:seqr@broadinstitute.org">contact us</Anchor>
+        </List.Item>
+        <List.Item>
+          Please use the <Anchor href="http://github.com/broadinstitute/seqr/issues">github issues page</Anchor> to
+          submit bug reports or feature requests
+        </List.Item>
```

Comment:
It seems that a list item is missing in the action list. You can find it at https://docs.google.com/document/d/1lCis6ze8FYGqI6AB9Fm4tqOrSJg3NMp_ddAiwOyxuVM/edit?disco=AAAAMEh6HVs

Reply:
that list item is no longer accurate so I intentionally removed it. That document is the proposal, this PR is what we actually want